packages/rainbowkit/package.json

```diff
@@ -45,7 +45,7 @@
     "ethers": ">=5.5.1",
     "react": ">=17",
     "react-dom": ">=17",
-    "wagmi": "0.9.x"
+    "wagmi": "0.11.x"
```

Comment:
Highly recommend changing this to `>=0.9` as to not force app developers to upgrade `wagmi` if they aren't ready to for some reason.